src/components/controllers/tutorial/style.module.css

```diff
@@ -1,20 +1,3 @@
-.errorOverlayWrapper {
-	position: absolute;
-	top: 0;
-	left: 0;
-	bottom: 0;
-	right: 0;
-	height: 100%;
-	display: flex;
-}
-
-.close {
-	position: absolute;
-	top: 1rem;
-	right: 1rem;
-	z-index: 10;
-}
-
```

Comment:
Most of these error styles weren't working correctly as they were sitting under the REPL Runner's styles and/or were redundant, so I stripped them out. Errors should continue to look as they have in the REPL & Tutorial.